### PR TITLE
Add remove action

### PR DESCRIPTION
**Pull request by ammarkandel**
# Milestone 3
## Add Actions to the book store app :
- Add book action.
- Remove book action.

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @ammarkandel, great job on this milestone :fireworks: Awesome job using the recommended linter configuration file :muscle: There are a few comments we need to address before proceeding. Kindly check the comments below:

- Kindly implement the handleChange functionality in the BookForm component and attach it to the input's and select's onChange event
- `BookList` - Kindly implement the handleRemoveBook functionality
- Kindly connect the removeBook action in the BookList component
- Kindly pass the handleRemoveBook functionality to the Book component
- The Book component should stay presentational, meaning the Book component should be a stateless component but currently it is a stateful component. Let us fix this
- Also our project directory can be improved, let us create a new directory, "Container" and put all our stateful files/components in it. The "Container" directory should have files that interact with the state of the application
- Let us improve our README file but adding commands about how to install the dependencies, because without installing them, the "npm start" or "yarn start" commands will not work

[Optional]
- Try to make a lot of commits going forward

Once these are addressed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: src/components/containers/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// import classes from './BooksList.module.css';
```
Comment (bellom) [inline src/components/containers/BooksList.js]: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @ammarkandel   :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

# Status: Changes Required! ♻️ 

_I will advise you to focus on this milestone 3, one step at a time, make sure to read carefully, and follow the instructions in the pathwright to save you from multiple CRR._

_I wish I could comment directly on your codebase but the new update code is not on file change._

**You still need to work on some comments from the previous review;**

**1.** Kindly implement the handleChange functionality in the `BookForm` component and attach it to the input's and select's onChange event.
example: 

Create the handleChange function then attach it to input.

```
      <input
        onChange={handleChange}
      />
```
2. Kindly pass the handleRemoveBook functionality to the Book component;
image
example;
`import { removeBook } from '../actions/index'`
`const BookList = ({ books, removeBook }) => {...`

**3.** File structure: It should look like this `src/container` and `src/component`.

**4.**  Update your README file.
We also need inform users about these script in the README file. This information can be placed in the Usage section. This way users can know the command to run to start your application.
**Let's make your repo look more professional by updating your `README` file**.
Checkout out the below link for help, feel free to follow this [Microverse template](https://github.com/microverseinc/readme-template) from pathwright.


**And also, check my direct comments below;**


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @ammarkandel, congratulations on this milestone :fireworks: Awesome. The handleChange method is implemented and it is attached to the inputs and selects onChange event :muscle: The file structure now looks professional :1st_place_medal: The handleRemoveBook method is implemented :100: 

[Optional]
- Let us kindly make our README professional, that is the first thing anyone looks at before looking at the code. So going forward, let us make it right by looking at the template the previous reviewer sent in the review. Add `npm install or yarn install`


I hereby approve this milestone :+1: 
Kindly go ahead and merge into development and continue with the next milestone :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**